recipes/app3.rb

```diff
@@ -138,3 +134,31 @@
   ]
   sensitive true
 end
+
+[
+  ['etherpad-lite.osuosl.org', 8085, 'osuosl/etherpad', data_bag_item('etherpad', 'mysql_creds_osl')],
+  ['etherpad-snowdrift.osuosl.org', 8086, 'osuosl/etherpad-snowdrift', data_bag_item('etherpad', 'mysql_creds_snowdrift')],
+].each do |hostname, port, image, creds|
+  # Check if attribute is set for testing
+  db_host = node['osl-app'].attribute?('db_hostname') ? node['osl-app']['db_hostname'] : creds['db_hostname']
+
+  docker_image image do
+    tag 'latest'
```

Comment:
Can you please use the new tag I created for each. Keep in mind this tag might be different between each of them. It might be better to just skip the hash loop and do the resources individually so it's easier to read.